packages/backend/src/utils/modelsUtils.spec.ts

```diff
@@ -91,7 +135,7 @@ describe('getRemoteModelFile', () => {
       },
     } as unknown as ModelInfo);
 
-    expect(path).toBe(`${MACHINE_BASE_FOLDER}dummy.guff`);
```

Comment:
Since the trailing slash has been removed, the test was failing, but it was nonsense to concat manually, let's use posix.join has it is the remoteModelFile